boot/bootutil/src/loader.c

```diff
@@ -1039,6 +1040,12 @@ boot_go(struct boot_rsp *rsp)
 
     switch (swap_type) {
     case BOOT_SWAP_TYPE_NONE:
+#ifdef BOOTUTIL_VALIDATE_SLOT0
```

Comment:
Well -DSYMBOL should define it to 1, so it usually works. But ifdef is more common. I'm not sure if you wanted to use the Newt specific configuration macro.